ssh/tailssh/incubator.go

```diff
@@ -262,6 +262,7 @@ func startWithPTY(cmd *exec.Cmd, ptyReq ssh.Pty) (ptyFile *os.File, err error) {
 	updateStringInSlice(cmd.Args, "--has-tty=false", "--has-tty=true")
 	if ptyName, err := ptyName(ptyFile); err == nil {
 		updateStringInSlice(cmd.Args, "--tty-name=", "--tty-name="+ptyName)
+		cmd.Env = append(cmd.Env, fmt.Sprintf("SSH_TTY=/dev%s", ptyName))
```

Comment:
Is this missing a slash between dev and %s?

Reply:
No, but it's cleaner to use `filepath.Join` here.